internal/operator/clusterutilities.go

```diff
@@ -735,6 +737,51 @@ func GetPgbackrestS3EnvVars(cluster crv1.Pgcluster, clientset *kubernetes.Client
 	} else {
 		s3EnvVars.PgbackrestS3Region = Pgo.Cluster.BackrestS3Region
 	}
+	if cluster.Spec.BackrestS3URIStyle != "" {
+		s3EnvVars.PgbackrestS3URIStyle = cluster.Spec.BackrestS3URIStyle
+	} else {
+		s3EnvVars.PgbackrestS3URIStyle = Pgo.Cluster.BackrestS3URIStyle
+	}
+	if cluster.Spec.BackrestS3VerifyTLS != "" {
+		s3EnvVars.PgbackrestS3VerifyTLS = cluster.Spec.BackrestS3VerifyTLS
+	} else {
+		s3EnvVars.PgbackrestS3VerifyTLS = Pgo.Cluster.BackrestS3VerifyTLS
+	}
+
+	// if set, pgBackRest URI style must be set to either 'path' or 'host'. If it is neither,
+	// log an error and stop the cluster from being created.
+	if s3EnvVars.PgbackrestS3URIStyle != "path" && s3EnvVars.PgbackrestS3URIStyle != "host" &&
+		s3EnvVars.PgbackrestS3URIStyle != "" {
+		log.Error("pgBackRest S3 URI style must be set to either \"path\" or \"host\".")
+		return ""
+	}
+
+	// If the pgcluster has already been set, either by the PGO client or from the
+	// CRD definition, parse the boolean value given.
+	// If this value is not set, then parse the value stored in the default
+	// configuration and set the value accordingly
+	var verifyTLS bool
+	var err error
+	if cluster.Spec.BackrestS3VerifyTLS != "" {
+		verifyTLS, err = strconv.ParseBool(cluster.Spec.BackrestS3VerifyTLS)
+		if err != nil {
+			log.Error(err.Error())
+			return ""
+		}
+	} else {
+		verifyTLS, err = strconv.ParseBool(Pgo.Cluster.BackrestS3VerifyTLS)
+		if err != nil {
+			log.Error(err.Error())
+			return ""
+		}
+	}
```

Comment:
I would recommend simplifying:

```suggestion
	verifyTLS, _ := strconv.ParseBool(Pgo.Cluster.BackrestS3VerifyTLS)
	
	if cluster.Spec.BackrestS3VerifyTLS != "" {
		verifyTLS, _ = strconv.ParseBool(cluster.Spec.BackrestS3VerifyTLS)
	}
```

Reply:
that works for me. Given I'm default to 'false' in other places, that makes sense. Updated.